core/process.py

```diff
@@ -890,6 +892,21 @@ def JobWait(self, waiter):
         """Wait for this process/pipeline to be stopped or finished."""
         raise NotImplementedError()
 
+    def InForeground(self):
+        # type: () -> bool
+        """Returns true if this job is running in the foreground."""
+        return self.in_foreground
```

Comment:
maybe this getter is a bit much and using `.in_foreground` directly might be OK? do you have a pattern/preference for this @andychu?